src/components/m-datepicker/index.js

```diff
@@ -5,17 +5,23 @@ import styles from './index.scss';
 // import the template used for this component
 import template from './_template';
 
-console.log('\n🚫 Datepicker is not ready to be used. Coming soon 🚫\n');
+import Datepicker from './js/datepicker';
+
+import getAttribute from '../../js/get-attribute';
 
 class AXADatepicker extends BaseComponentGlobal {
-  static tagName = 'axa-datepicker'
+  static tagName = 'axa-m-datepicker'
 
   // Specify observed attributes so that attributeChangedCallback will work,
   // this is essential for external re-rendering trigger.
-  static get observedAttributes() { return ['classes', 'button-ok', 'button-cancel']; }
+  static get observedAttributes() {
+    return ['classes', 'button-ok', 'button-cancel', 'locale', 'value', 'selected-index', 'start-year', 'start-month', 'lower-end-year', 'higher-end-year', 'output-iso'];
```

Comment:
`selected-index` of what? year, month, both?

Reply:
it was needed for the datepicker-body. Here is wrong. @NoahLiechti please remove it